### PR TITLE
new design added

### PR DESCRIPTION
**Pull request by gnomjogson**
new design added by RU.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,99 +1,48 @@
-baseURL: https://dev.codefor.de
+baseURL: https://dev.rgb3000.com
```
Comment (ubergesundheit) [inline config.yaml:1]:
```suggestion
baseURL: https://dev.codefor.de
```

Comment (gnomjogson) [reply config.yaml]: fixed

File: themes/codefor-theme/layouts/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            <ul class="list-unstyled">
+              <li><a href="https://github.com/okfde"><img src="/img/github.svg" class="img-fluid footer-social-icon" alt="Github"></a></li>
+              <li><a href="https://twitter.com/codeforde"><img src="/img/twitter.svg" class="img-fluid footer-social-icon" alt="twitter"></a></li>
+            </ul>
+          </div>
+    
+          <div class="col-24 col-sm-12 col-md-6 col-lg-6 mt-3 mt-sm-5 mt-lg-0 ">
+            <h4>Ein Projekt der</h4>
+            <ul class="list-unstyled">
+              <li><a href=""><img class="img-fluid" style="width: 180px;margin-left:-10px;" src="/img/open-knowledge.svg" alt="Open Knowledge Foundation"></a></li>
```
Comment (wagnerpeer) [inline themes/codefor-theme/layouts/partials/footer.html:68]: Missing/incorrect `href=""`.
I think it should be `href="https://okfn.de"`

Comment (ubergesundheit) [reply themes/codefor-theme/layouts/partials/footer.html:68]:
```suggestion
              <li><a href="https://okfn.de"><img class="img-fluid" style="width: 180px;margin-left:-10px;" src="/img/open-knowledge.svg" alt="Open Knowledge Foundation"></a></li>
```

**Pull request merged**